components/optimise-cv/OptimiseCV.tsx

```
@@ -649,13 +734,16 @@ export const OptimiseCV = () => {
         >
           <div className="max-w-3xl mx-auto px-6 py-4 bg-gray-50 dark:bg-gray-800/50 rounded-lg text-sm text-gray-600 dark:text-gray-400">
             <p className="mb-2">
-              Our AI-powered CV optimisation is designed to provide valuable insights, but results may vary. We're always working to improve—if you spot anything that seems off, let us know at{' '}
-              <a 
-                href="mailto:feedback@apprenticewatch.com" 
+              Our AI-powered CV optimisation is designed to provide valuable
+              insights, but results may vary. We're always working to improve—if
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)